portal-web/test/functional/com/liferay/portalweb/macros/Upgrade.macro

```diff
@@ -5054,6 +5054,34 @@ definition {
 		Portlet.viewNoPermissionPG();
 	}
 
+	macro viewRemoteApp {
+		task ("Assert Remote App table entry") {
+			RemoteApps.goToRemoteAppsPortlet();
+
+			RemoteApps.viewTableEntries(
+				entryName = "${upgradeName}",
+				entryType = "${upgradeType}");
+		}
+
+		task ("Assert Remote App Name, Type, and URL fields") {
+			Click(
+				key_tableEntryName = "${upgradeName}",
+				locator1 = "RemoteApps#TABLE_ENTRY_NAME_REMOTE_TABLE");
+
+			AssertTextEquals(
+				custom_entryName = "${upgradeName}",
+				key_text = "Name",
+				locator1 = "TextInput#ANY",
+				value1 = "${upgradeName}");
+
+			AssertTextEquals(
+				custom_entryURL = "${upgradeURL}",
+				key_id = "iFrameURL",
+				locator1 = "RemoteAppsEntry#URL",
+				value1 = "${upgradeURL}");
+		}
+	}
+
```

Comment:
1. Let's split the two tasks into two macros
2. If we're asserting something, then the name of the macro with assert instead. Let's start avoiding view when adding new macros. I know other Remote App macros still use view and use assert functions and they'll need to be refactored but we need to move away from that pattern. So something like: assertRemoteAppEntry and assertRemoteAppEntryFields
3. Do not add component specific macros in Upgrade.macro. Create a new macro file RemoteAppUpgrade.macro. See https://liferay.slack.com/archives/CD7939WBE/p1640688434222500?thread_ts=1640688403.222400&cid=CD7939WBE